workflow/libero_flow_poc/docker-compose.yml

```diff
@@ -0,0 +1,88 @@
+version: '3'
+
+services:
+  web:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    command: >
+            /bin/bash -c "
+                until echo > /dev/tcp/postgres/5432; do sleep 1; done
+                python manage.py migrate && python manage.py runserver 0.0.0.0:8000
+            "
+    env_file:
+      - '.env'
+    ports:
+      - '8000:8000'
+    depends_on:
+      - postgres
+      - redis
+      - rabbitmq
+
+  decider:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    working_dir: /srv/app
+    command: >
+            /bin/bash -c "
+                until echo > /dev/tcp/rabbitmq/5672; do sleep 1; done
+                python run_decider.py
+            "
+    depends_on:
+      - rabbitmq
+
+  scheduler:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    working_dir: /srv/app
+    command: >
+            /bin/bash -c "
+                until echo > /dev/tcp/rabbitmq/5672; do sleep 1; done
+                python run_scheduler.py
+            "
+    depends_on:
+      - rabbitmq
+
+  worker:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    working_dir: /srv/app
+    command: >
+            /bin/bash -c "
+                until echo > /dev/tcp/rabbitmq/5672; do sleep 1; done
+                python run_worker.py
+            "
+    depends_on:
+      - rabbitmq
+
+  redis:
+    image: 'redis:3.0-alpine'
+    ports:
+      - '6379:6379'
+    volumes:
+      - 'redis:/data'
```

Comment:
I didn't understand what Redis was used for?

Reply:
The session layer to store workflow instance values, though I haven't committed the `session_store` code yet 👾 as still tweaking.